state-reconstruct-fetcher/src/l1_fetcher.rs

```diff
@@ -546,6 +548,14 @@ impl L1Fetcher {
     }
 }
 
+fn make_support(url: String) -> Box<dyn BlobSupport + Send + Sync> {
+    if url.starts_with("https://blobscan.com") {
+        Box::<ScrapingSupport>::default()
```

Comment:
The scraping doesn't use a custom URL, right? It only operates on a static one. So, maybe we should actually add a flag in the CLI instead?

Reply:
Well, we could, but extra command-line arguments don't look so ephemeral to me...

Reply:
Yeah, the earlier idea that I had in mind was to hardcode the scraper implementation as the set one. I guess the question is do we need to be able to conveniently switch between these implementations? If not, we automatically remove the need for this method as well as the CLI-argument alternative. And, since we can track the blobscan issue ourselves, I would say that there's no strong need to do so. We'll retouch this code once that's resolved anyways, so in the end I don't have a strong opinion.

Reply:
Well, I'd like to know whether we can (finally) finish the sync using the blobscan.com GUI - and if we can't, then how the API works on the blobs that cannot be retrieved through the GUI... That suggests using a single program that can do both - as I said above, it might not be strictly necessary, but it surely simplifies debugging when I have a single program to debug... :-)